spec/transaction/components/schemas.yaml

```diff
@@ -471,6 +471,24 @@ components:
             description: The base64-encoded 128-bit MD5 digest of the associate part data.
             type: string
             example: yLLiDqX2OL7mcIMTjob60A==
+    update_interval:
+      description: |
+        Duration in seconds until the next update of the data is expected
+
+        Regularly updated asset objects need different cache settings than the values
+        set by default (7200 seconds = 2 hours). The `update_interval` parameter has
+        to be set to notify the system on when the next update of this asset object is
+        expected so that cache headers can be set correctly by the system:
+        - `update_interval > 0` defines the interval in seconds until the
+        next update will be sent to the system (maximum is 1 week = 604800 seconds)
+        - `update_interval = 0` means the data can be updated anytime in the future,
```

Comment:
Currently I implemented that `0` sets never cache, with this docu it might be tempting for data that as non regular update to use this value (interval from 1s to 1week for example) and this would mean that no cache would be set for data that might not change for a week !
Maybe we should document that the `update_interval` should be the minimal update interval and that `update_interval = 0` should only be used for real time data. I mean for the use case were the data owner knows that its data has un-regular update at a minimum interval of 1h it would be best to use a cache settings of a few minutes than no cache !

Reply:
Docu adapted